angular-example/src/app/modules/shared/arkose/arkose.component.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @angular-eslint/no-output-on-prefix */
```

Comment:
Angular recommends that you don't prefix outputs with `on`. I think this situation is fine due to the nature of how the enforcement props are named, hence this comment to ignore this linting.